packages/amplify-cli/src/plugin-manager.ts

```diff
@@ -42,8 +47,10 @@ export async function getPluginPlatform(): Promise<PluginPlatform> {
 function isCoreMatching(pluginPlatform: PluginPlatform): boolean {
   try {
     const currentCorePluginDirPath = getCorePluginDirPath();
```

Comment:
This could be combined into a single expression.

Reply:
same reason as above, I think it's easier to read and maintain to explicitly lay out the code like this.